src/logic/map_objects/tribes/ship.cc

```diff
@@ -439,6 +452,19 @@ void Ship::ship_update_expedition(Game& game, Bob::State& /* state */) {
 			             _("An expedition ship found a new port build space."),
 			             "images/wui/editor/fsel_editor_set_port_space.png");
 		}
+	} else if (ship_state_ == ShipStates::kExpeditionPortspaceFound) {
+		if (!expedition_->seen_port_buildspaces.empty()) {
```

Comment:
I don't think it should ever be empty. Maybe log it if it happens anyway?

Reply:
Well I simply was not sure whether there might be a cornercase for it to be empty. e.g. a race between construct and periodic check. so i tried to safeguard the thing without throwing an error.